Makefile

```diff
@@ -170,3 +171,15 @@ pao-functests-only:
 cluster-clean-pao:
 	@echo "Cleaning up performance addons artifacts"
 	hack/clean-deploy.sh
+
+# Performance Profile Creator (PPC)
+.PHONY: build-performance-profile-creator
+build-performance-profile-creator: 
```

Comment:
Should I add this target as a dependency to "build" target to ensure PPC is built with the default target or do you prefer to keep as it is right now to make PPC build explicit?

Reply:
IMHO, we should build this in all the cases we build PAO, this probably means to add it at dependency of the `build` target. But let's hear from @jmencak .

Reply:
We can add this to the `build` target.  This will not slow down the entire build much.  If/when it does, we should look into cleaning up the `Makefile` and only build this if its source files change.

Reply:
Added `build-performance-profile-creator` target as `build` target dependency